course6/toyapp_mikhail/displaylib/surface.cc

```diff
@@ -1,37 +1,90 @@
 #include "surface.h"
 
 #include <cassert>
+#include <cmath>
+#include <utility>
 
 namespace Display {
 
-Surface::Surface(int w, int h)
-{
-	mSurface = SDL_CreateRGBSurface(0, w, h, 32, 0, 0, 0, 0);
-}
+	Surface::Surface(int w, int h)
```

Comment:
see: by messing with whitespace you've created a large, whitespace-only change in the patch. It creates a noise and makes reviewing harder.